src/libraries/System.Private.CoreLib/src/System/Globalization/GlobalizationMode.cs

```diff
@@ -13,7 +13,9 @@ internal static partial class GlobalizationMode
         private static partial class Settings
         {
             internal static bool Invariant { get; } = AppContextConfigHelper.GetBooleanConfig("System.Globalization.Invariant", "DOTNET_SYSTEM_GLOBALIZATION_INVARIANT");
-#if TARGET_MACCATALYST || TARGET_IOS || TARGET_TVOS || TARGET_BROWSER
+#if TARGET_MACCATALYST || TARGET_IOS || TARGET_TVOS
+            internal static bool Hybrid { get; } = true;
```

Comment:
This does not look right. We ignore the user's setting now if they would like to disable HG (keep in mind that while setting HG as default it has certain differences in behavior that users might not want to accept and they would prefer to switch HG off). Use `DOTNET_SYSTEM_GLOBALIZATION_HYBRID` to propagate the MSBuild setting to managed code.

Reply:
This is done intentionally as we were facing issue with some FunctionalTests.
This PR does not allow for iOS to have Hybrid as switch, instead it will work only with Hybrid mode.
The reason is in runtime we need to duplicate a lot of files and build 2 runtimes (1. Loading ICU libs 2. Hybrid using system ICU) to be able to have Hybrid mode as switch. We decided for now to make changes the way to have only one runtime with Hybrid mode.